script.module.kodi-six/libs/kodi_six/__init__.py

```diff
@@ -5,7 +5,7 @@
 Wrappers around Kodi Python API that normalize byte and Unicode string handling
 """
 
-from __future__ import absolute_import
+from __future__ import absolute_import, division, unicode_literals
```

Comment:
Why unnecessary imports? `unicode_literals` I can understand although non-ASCII literals are not used in the code, but why `division`?

Reply:
I agree that they are unnecessary, however if you don't do this, users that include **kodi-six** for testing (like we do) have to disable these messages project-wide in tox. And there's no real difference whether you add division or not.

Reply:
Not only are these unnecessary, once you stop supporting Python 2, they can be removed altogether ;-) So it is a temporary measure by all means.

Reply:
Anyway, I see no point in adding unused code. Please remove those.